types/mui-image/index.d.ts

```diff
@@ -29,6 +29,7 @@ interface MuiImageProps {
     width?: React.CSSProperties['width'] | number | undefined;
     wrapperClassName?: string | undefined;
     wrapperStyle?: React.CSSProperties | undefined;
+    title?: string;
```

Comment:
should be `title?: string | undefined` for TS4.4 [exact optional property types](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-4.html#exact-optional-property-types---exactoptionalpropertytypes) support